sdk/iothub/Azure.ResourceManager.IotHub/src/autorest.md

```diff
@@ -95,6 +102,7 @@ format-by-name-rules:
   '*Uris': 'Uri'
   'thumbprint': 'any'
   'certificate': 'any'
+  'UserAssignedIdentity': 'arm-id'
```

Comment:
I did not see a change related with this, maybe it does not take effect

Reply:
updated, forget to regen the api file